tests/io/test_mdv_grid.py

```diff
@@ -212,7 +212,7 @@ def test_mdv_degree_grid():
     grid = pyart.io.read_grid_mdv(
         pyart.testing.MDV_GRID_FILE, file_field_names=True)
 
-    assert 'refl' in grid.fields.keys()
+    assert 'ref' in grid.fields.keys()
```

Comment:
Same here - `refl` not `ref`

Reply:
Okay, that should be all the `ref` to `refl` now... Not sure why those changes weren't pulled when I merged changes from main